src/main/java/Duke.java

```diff
@@ -1,10 +1,114 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static final int DEADLINE_LENGTH = 8;
+    public static final int EVENT_LENGTH = 5;
+
+    private static Task[] tasks = new Task[100];
```

Comment:
Try to avoid the use of a magic number here. Using a named constant makes the code easier to understand because the name tells us more about the meaning of the number.